templates/master.erb

```diff
@@ -1,29 +1,13 @@
-# ZooKeeper URL (used for leader election amongst masters)
-ZK="<%= @zookeeper %>"
+MESOS_work_dir="<%= @work_dir %>"
+MESOS_zk="<%= @zookeeper %>"
+MESOS_quorum=<%= @quorum %>
+MESOS_ip="<%= @listen_address %>"
+MESOS_port=<%= @master_port %>
+MESOS_cluster="<%= @cluster %>"
+MESOS_log_dir="<%= @log_dir %>"
+MESOS_logbufsecs=1
+MESOS_logging_level="INFO"
```

Comment:
I don't like default that you can't change. Though, in this case you could override this value in `@env_var` with same key.